caffe2/core/nomnigraph/test.cc

```diff
@@ -0,0 +1,692 @@
+#include "nomnigraph/Graph/Algorithms.h"
+#include "nomnigraph/Graph/Graph.h"
+
+#include "nomnigraph/Converters/Caffe2.h"
+#include "nomnigraph/Converters/Dot.h"
+
+#include "nomnigraph/Transformations/ConnectNet.h"
+#include "nomnigraph/Transformations/OperatorFusion.h"
+#include "nomnigraph/Transformations/Match.h"
+
+#include "nomnigraph/Support/Casting.h"
+
+#include <fstream>
+#include <iomanip>
+#include <stdio.h>
+
+#define ADD_ARG(_op, _name, _type, _val)                                       \
```

Comment:
You could get away with just two overloaded functions to add s(tring) and i(nt) arguments